cloudbuild-trigger-cd.yaml

```diff
@@ -28,17 +28,17 @@ steps:
   args:
   - 'build'
   - '-t'
-  - 'gcr.io/$PROJECT_ID/hello-cloudbuild:$SHORT_SHA'
+  - 'us-central1-docker.pkg.dev/$PROJECT_ID/my-repository/hello-cloudbuild:$SHORT_SHA'
   - '.'
 
-# This step pushes the image to Container Registry
+# This step pushes the image to Artifact Registry
 # The PROJECT_ID and SHORT_SHA variables are automatically
 # replaced by Cloud Build.
 - name: 'gcr.io/cloud-builders/docker'
```

Comment:
Should the name change as well since gcr is Container Registry's domain?

Reply:
Thank you so much for the quick review, @shannonxtreme! ⚡

The `name` here is referring to the Cloud Build "builder image" ([relevant doc here](https://cloud.google.com/build/docs/build-config-file-schema#name)) that gets executed at this step.
We can use any publicly available image here, but `gcr.io/cloud-builders/docker` is an [official Cloud Build builder image](https://cloud.google.com/build/docs/cloud-builders#supported_builder_images_provided_by) — so I'm guessing it's best practice to stick to it.
The image is maintained at 
[github.com/GoogleCloudPlatform/cloud-builders](https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/docker). 

As you suggested, it would be nice to use an Artifact Registry-hosted alternative here (just to be consistent). I found a Google-internal [Google Doc](https://docs.google.com/document/d/15iIqI3W7IoZrRInmrf2TALr6jN7rHXfg_SVpcOYbQiw/edit#heading=h.xzptrog8pyxf) and [page](https://g3doc.corp.google.com/cloud/build/g3doc/internal/ARBuilders.md?cl=head) talking about a relevant project — but it seems like an ongoing project.

Hi @bendory, just a quick question: does the `gcr.io/cloud-builders/docker` image currently have an Artifact Registry-hosted alternative? :)

Reply:
I've emailed @bendory (David).
David informed me that we do not currently have Artifact Registry-hosted alternatives for the [official Cloud Build builder images](https://cloud.google.com/build/docs/cloud-builders#supported_builder_images_provided_by).
I've forwarded the email to you, @shannonxtreme, for visibility.

I'm resolving this conversation.
Thanks, you two!